partiql-parser/src/lexer/embedded_ion.rs

```diff
@@ -0,0 +1,136 @@
+use crate::error::LexError;
+use crate::lexer::{CommentLexer, SpannedResult};
+use logos::{Logos, Span};
+use partiql_common::syntax::line_offset_tracker::LineOffsetTracker;
+use partiql_common::syntax::location::ByteOffset;
+use std::borrow::Cow;
```

Comment:
```
 GitHub Actions / clippy

unused import: `std::borrow::Cow`

warning: unused import: `std::borrow::Cow`
 --> partiql-parser/src/lexer/embedded_ion.rs:6:5
  |
6 | use std::borrow::Cow;
  |     ^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

Reply:
removed; not sure why clippy didn't automatically...